### ISSUE TITLE
External Integrations not working?

### ISSUE THREAD

**Issue by RandomBloke85** [navidrome/navidrome#1263]
Hello

I installed the Navidrome freeBSD pkg (0.40.0-FREEBSD (8799358)) on TrueNas a couple of days ago and running it behind a nginx reverse-proxy. Everything is running fine and I love Navidrome, since it is extremly fast, easy to install and does what it is supposed to do - serving music. So thank you for that!

However, I tried to integrate the external integrations :-) as described here https://www.navidrome.org/docs/usage/external-integrations/ in the freeBSD section. But neither Dsub, substreamer nor Ultrasonic (all Android) show any pictures or Artist-bios. When I have the last.fm option activated, it makes the server stop. So I deactiveted last.fm, left spotify active, but it shows no Artist pics.

Not a functional issue, just cosmetics. Very happy with the server. Thank you for your time and effort.

Regards

**Comment by metalheim** [comment]
Can you check the logs? And put the results here? 

Usually they will show errors when something is not working. Especially when something is breaking the server.

If possible, please censor your spotify/last fm api keys. Later versions of navidrome will automatically censor it, but I think v0.40 does not have that feature.

**Comment by RandomBloke85** [comment]
Could you tell which logs that would be and where to find them? I guess they should be in /var/log. But the debug.log shows only some flacs it couldn't read. And that's not a Navidrome issue. Sorry, I'm not so sophisticated...

**Comment by deluan** [comment]
If you installed using the instructions from the docs, you can find the logs here: https://www.navidrome.org/docs/faq/#where-are-the-logs

**Comment by travisfalk** [comment]
Thanks for the great software.
I'm having the same problem. No artist Bio's. Everything else appears to work ok.
Last.fm similar artists are populated properly so the api appears to be working.
The error I get is 'error="json: cannot unmarshal number into Go struct field .artist.stats.listeners of type string" mbid= requestId=6e22a2f673f8/54AvCo8uKq-000005'
I'm using a docker container on Ubuntu server 20.04.
Attached a log for your viewing pleasure.
[Travs_Navidrome_Log.txt](https://github.com/navidrome/navidrome/files/6912027/Travs_Navidrome_Log.txt)

**Comment by travisfalk** [comment]
Looking back through other issues, this has probably been already addressed in issue #1246

**Comment by deluan** [comment]
@travisfalk Yes, the issue you are facing is already fixed and will be part of the next release (this week?)

**Comment by Kiina** [comment]
I'm not getting any errors or even a mention of spotify in the logs except for `{"log":"time=\"2021-08-17T00:05:31Z\" level=debug msg=\"Spotify integration is ENABLED\"\n","stream":"stderr","time":"2021-08-17T00:05:31.189105845Z"}` (but it also does that with the lastfm one I haven't configured). Tried switching over to the develop tag of the docker image. When I do a `printenv` inside the container everything seems correct:

```
HOSTNAME=ba452efe0bb5
SHLVL=1
HOME=/
GODEBUG=asyncpreemptoff=1
ND_SESSIONTIMEOUT=24h
ND_MUSICFOLDER=/music
TERM=xterm
ND_PORT=4533
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
ND_SPOTIFY_ID=redacted
ND_SPOTIFY_SECRET=redacted
ND_LOGLEVEL=error
ND_BASEURL=
ND_SCANSCHEDULE=1h
ND_DATAFOLDER=/data
PWD=/app
```
